merge.go

```diff
@@ -192,8 +192,11 @@ func persistMergedRest(segments []*SegmentBase, dropsIn []*roaring.Bitmap,
 	fieldDvLocsStart := make([]uint64, len(fieldsInv))
 	fieldDvLocsEnd := make([]uint64, len(fieldsInv))
 
-	tfEncoder := newChunkedIntCoder(uint64(chunkFactor), newSegDocCount-1)
-	locEncoder := newChunkedIntCoder(uint64(chunkFactor), newSegDocCount-1)
+	// these int coders are initialized with chunk size 1024
+	// however this will be reset to the correct chunk size
+	// while processing each individual field-term section
+	tfEncoder := newChunkedIntCoder(1024, newSegDocCount-1)
```

Comment:
any reasons for the hardcoded numeric value than the default/legacy constants already made?

Reply:
It could be, as the comment above describes it's unfortunate the way the API was designed, as we never this value at all.  I would use 0, or some other value if I could make it more clear.  I felt using that constant instead of the value would further mislead someone into thinking we used that actual value.